cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1020,6 +1020,22 @@
                 :cylc:conf:`flow.cylc[runtime][<namespace>]platform` matches
                 the name of this platform group.
             ''')
+            with Conf('selection'):
+                Conf(
+                    'method', VDR.V_STRING, default='random',
+                    options=['random', 'definition order'],
+                    desc='''
+                        Host selection method for the platform. Available
+                        options:
```

Comment:
Isn't this platform selection rather than host selection?

```suggestion
                        Method for selecting the platform from the group.
                        
                        Options:
```